packages/flutter/test/material/menu_anchor_test.dart

```diff
@@ -945,6 +945,48 @@ void main() {
       expect(material.color, equals(Colors.red));
     });
 
+    testWidgets('make clipBehavior be set correctly for MenuAnchor', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        MaterialApp(
+          home: Material(
+            child: Center(
+              child: MenuAnchor(
+                clipBehavior: Clip.antiAlias,
+                menuChildren: const <Widget> [
+                  MenuItemButton(
+                    child: Text('Button 1'),
+                  ),
+                  MenuItemButton(
+                    child: Text('Button 2'),
+                  ),
+                  MenuItemButton(
+                    child: Text('Button 3'),
+                  ),
+                ],
+                builder: (BuildContext context, MenuController controller, Widget? child) {
+                  return FilledButton(
+                    onPressed: () {
+                      if (controller.isOpen) {
+                        controller.close();
+                      } else {
+                        controller.open();
+                      }
+                    },
+                    child: const Text('Tap me'),
+                  );
+                },
+              ),
+            )
+          )
+        ),
+      );
+
+      await tester.tap(find.text('Tap me'));
+      await tester.pump();
+      final Material material = getMenuBarMaterial(tester);
+      expect(material.clipBehavior, equals(Clip.antiAlias));
```

Comment:
Can you please test the default clip behavior first then the custom clip behavior

Also, please add test comments, something like:
```dart
// Test default clip behavior.
...


// Test custom clip behavior.
...
```

Reply:
Done

Reply:
This should be a single test. 

Firstly, don't pass any custom behavior, check the default clip behavior, follow by custom behavior, and check.

These two comments are in a single test. Sorry, if it wasn't clear.

Reply:
I'm not sure how to do this with only a single widget. How do I set the property of a widget if I'm initially testing the default behaviour? Looked online but couldn't find anything helpful.